data/strings/types_strings.txt

```diff
@@ -27719,12 +27760,12 @@
     nl = Sloot
     pt = Vala
     pt-BR = Vala
-    ru = Канава
+    ru = Ров
     tr = Hendek
     uk = Канава
     zh-Hans = 水沟
 
-  [type.waterway.ditch.tunnel]
+  [type.natural.water.ditch]
     ref = type.waterway.ditch
     en = Culvert
     de = Durchlass
```

Comment:
please remove all translations here (let the `ref` take them from waterway.ditch) as they refer to the tunnel/culvert

Reply:
oops, i think i accidentally pasted that 😅 fixed